config/default/certmanager_metrics_manager_patch.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          volumeMounts:
+            - mountPath: /tmp/k8s-metrics-server/metrics-certs
+              name: metrics-certs
+              readOnly: true
+      volumes:
+        - name: metrics-certs
```

Comment:
`metrics-certs` name in Certificate will be prefixed to `lws-metrics-certs`. Does the prefixing mechanism will prepend `lws` to here as well?

Reply:
nope, no prefix here